tutorials/circuits/2_plotting_data_in_qiskit.ipynb

```diff
@@ -7,715 +7,42 @@
     "# Qiskit Visualizations"
```

Comment:
<p>It will be nice to put plot_histogram and the others within backquotes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>